content/events/launchpad-social-ethcc.md

```diff
@@ -0,0 +1,35 @@
+---
+eventName: "Launchpad Social" #the name of your track or event and its mandatory
```

Comment:
@laeccles Its always suggested better to remove comments wherever present (any content following #)

Reply:
@WinsCodes resolved